system-parachains/people/people-polkadot/src/bridge_to_bulletin_config.rs

```diff
@@ -0,0 +1,280 @@
+}
+
+/// Proof of messages, coming from Polkadot Bulletin chain.
+pub type FromPolkadotBulletinMessagesProof<MI> =
+	FromBridgedChainMessagesProof<bp_polkadot_bulletin::Hash, LaneIdOf<Runtime, MI>>;
+/// Messages delivery proof for Polkadot Bridge Hub -> Polkadot Bulletin messages.
+pub type ToPolkadotBulletinMessagesDeliveryProof<MI> =
+	FromBridgedChainMessagesDeliveryProof<bp_polkadot_bulletin::Hash, LaneIdOf<Runtime, MI>>;
+
+/// Dispatches received XCM messages from other bridge.
+type FromPolkadotBulletinMessageBlobDispatcher = BridgeBlobDispatcher<
+	XcmRouter,
+	UniversalLocation,
+	BridgePolkadotToPolkadotBulletinMessagesPalletInstance,
+>;
+
+/// Transaction extension that refunds relayers that are delivering messages from the Polkadot
+/// Bulletin chain.
+pub type OnPeopleHubPolkadotRefundPolkadotBulletinMessages = BridgeRelayersTransactionExtension<
+	Runtime,
+	WithMessagesExtensionConfig<
+		StrOnPeopleHubPolkadotRefundPolkadotBulletinMessages,
+		Runtime,
+		WithPolkadotBulletinMessagesInstance,
+		RelayersForLegacyLaneIdsMessagesInstance,
+		PriorityBoostPerMessage,
+	>,
+	bp_messages::LegacyLaneId,
+>;
+bp_runtime::generate_static_str_provider!(OnPeopleHubPolkadotRefundPolkadotBulletinMessages);
+
+/// Add XCM messages support for PeopleHubPolkadot to support Polkadot->Polkadot Bulletin XCM messages.
+pub type WithPolkadotBulletinMessagesInstance = pallet_bridge_messages::Instance1;
+impl pallet_bridge_messages::Config<WithPolkadotBulletinMessagesInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo =
+		();
+
+	type ThisChain = bp_people_hub_polkadot::PeopleHubPolkadot;
+	type BridgedChain = bp_polkadot_bulletin::PolkadotBulletin;
+	type BridgedHeaderChain = BridgePolkadotBulletinGrandpa;
+
+	type OutboundPayload = XcmAsPlainPayload;
+	type InboundPayload = XcmAsPlainPayload;
+	type LaneId = LegacyLaneId;
+
+	type DeliveryPayments = ();
+	type DeliveryConfirmationPayments = ();
+
+	type MessageDispatch = XcmOverPolkadotBulletin;
+	type OnMessagesDelivered = XcmOverPolkadotBulletin;
+}
+
+/// Add support for the export and dispatch of XCM programs.
+pub type XcmOverPolkadotBulletinInstance = pallet_xcm_bridge_hub::Instance1;
+impl pallet_xcm_bridge_hub::Config<XcmOverPolkadotBulletinInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+
+	type UniversalLocation = UniversalLocation;
+	type BridgedNetwork = PolkadotBulletinGlobalConsensusNetworkLocation;
+	type BridgeMessagesPalletInstance = WithPolkadotBulletinMessagesInstance;
+
+	type MessageExportPrice = ();
+	type DestinationVersion = AlwaysV5;
+
+	type ForceOrigin = EnsureRoot<AccountId>;
+	// We don't want to allow creating bridges for this instance.
+	type OpenBridgeOrigin = EnsureNever<Location>;
+	// Converter aligned with `OpenBridgeOrigin`.
+	type BridgeOriginAccountIdConverter =
+		(ParentIsPreset<AccountId>, SiblingParachainConvertsVia<Sibling, AccountId>);
+
+	type BridgeDeposit = BridgeDeposit;
```

Comment:
We don't need deposit here
```suggestion
	type BridgeDeposit = ConstU128<0>;
```